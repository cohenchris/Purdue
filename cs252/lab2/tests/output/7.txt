0x564009c0d060
(nil)
[
	addr: 0096
	size: 3968
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x564009c0d0b0
(nil)
[
	addr: 0176
	size: 3888
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x564009c0d100
(nil)
[
	addr: 0256
	size: 3808
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x564009c0d150
(nil)
[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

