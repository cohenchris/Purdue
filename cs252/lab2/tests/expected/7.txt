
[
	addr: 0096
	size: 3968
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

[
	addr: 0176
	size: 3888
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

[
	addr: 0256
	size: 3808
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

[
	addr: 0336
	size: 3728
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

